LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2012-2020 Pivotal Software, Inc. All Rights Reserved.
```

Comment:
Just remove this line and I'll approve it ;)

Reply:
Done.